### DISCUSSION TITLE
Tshock 4.4.0 PreRelease 12 for Mobile

### DISCUSSION THREAD

**Discussion by bydioeds** [Pryaxis/TShock#2138] [Q&A] [answered]
Hello in advance thanks for answering, my question, is this something to be worried about? 

I'm testing the pre-release 12 using mobile and it seems to be working but i get these messages on terminal.

Server: mc.culturaotaku.xyz:33333, journey
As said i'm just testing.

![captura_3](https://user-images.githubusercontent.com/8415008/96617536-60ff4280-12c9-11eb-983f-def9869f0574.png)

**Comment by Kojirremer** [comment, marked as answer]
TShock doesn't support mobile yet. The forks that do support mobile are from third parties, and thus requests for support should be forwarded to them.

**Reply by bydioeds** [reply]
Oh, i see, i thought as there some parity this could work, thanks

**Comment by Terrabade** [comment]
Simply turn off debug mode in the config- that seems to be on by default in this build.

**Comment by Kyllingene** [comment]
Wow. It still impresses me that it is even _viable_ to try and host a server on your phone. Crazy tech these days!
